reco_utils/recommender/sar/sar_singlenode.py

```diff
@@ -70,6 +70,7 @@ def __init__(
         self.model_str = "sar_ref"
         self.user_affinity = None
         self.item_similarity = None
+        self.item_frequencies = None
```

Comment:
I think we can consider (in this PR or another one) to show the functionality of getting similar items in the deep dive notebook

Reply:
yeah, good idea